src/libopenrave-core/environment-core.h

```diff
@@ -2099,6 +2099,7 @@ class Environment : public EnvironmentBase
             }
         }
 
+        (*it)->ReleaseAllGrabbed();
         if( (*it)->IsRobot() ) {
```

Comment:
@AntoineRyu let's remove the IsRobot check. Thanks!

Reply:
I moved the `ReleaseAllGrabbed` out of the `IsRobot` check already based on our in person discussion. The remaining check is followed by a cast of `it` to `RobotBase`, so I think it makes sense to keep it.

Reply:
yes, you are right, thanks!